project/{{ cookiecutter.__folder_name }}/.github/workflows/backend.yml

```diff
@@ -1,124 +1,149 @@
 name: Backend CI
 
 on:
-  push:
-    paths:
-      - "backend/**"
-      - ".github/workflows/backend.yml"
-  workflow_dispatch:
-
-env:
-  IMAGE_NAME_PREFIX: {{ cookiecutter.__container_image_prefix }}
-  IMAGE_NAME_SUFFIX: backend
-  PYTHON_VERSION: "3.11"
+  workflow_call:
+    inputs:
+      base-tag:
+        required: true
+        type: string
+      image-name-prefix:
+        required: true
+        type: string
+      image-name-suffix:
+        required: true
+        type: string
+      python-version:
+        required: true
+        type: string
+      plone-version:
+        required: true
+        type: string
+
+defaults:
+  run:
+    working-directory: backend
 
 jobs:
 
-  meta:
+  lint:
```

Comment:
I would call it `check` instead of `lint` so it matches the Makefile commands.

Reply:
We decided against calling it check if I recall correctly, right @stevepiercy ?

Reply:
If it merely checks files for code quality, then `*check*` is an appropriate Makefile target, such as `make linkcheck`. Whatever it checks should be explicit.

In Volto, we chose `lint`, and provided a description of `Lint, or catch and remove problems, in code base`. See https://docs.google.com/spreadsheets/d/1C_kBeXrx5nRLYaLI8qNXhBOBD9j7Ty0r_DxJNDz7Cww/edit?gid=0#gid=0. But that is not clear to me whether it merely checks files or writes files.

If it actually writes files, then we want something else, because that is not a mere check.

Reply:
@stevepiercy we have `lint` (which only checks) and `format` (who writes too) in frontend_addon. I think that in the project too. We should unify the Makefile commands, I do not care about the CI jobs naming.

Reply:
@sneridagh so instead of changing the CI name here, we need to change the Makefile command to `lint`, correct? `check` is too vague of a command in Makefile.

Reply:
It is already there as `lint`. I've kept the `check` for compatibility reasons.